### DISCUSSION TITLE
Custom Installation Folder

### DISCUSSION THREAD

**Discussion by MaxMeier98** [flybywiresim/aircraft#3278] [Help]
Dear folks, 

I've been trying to install the mod several times now, but unfortunately it does not work. 

My Installation has been somewhat tricky. I've received the DVD-Box as a gift, thus I first tried to install the game via DVD. However, it took a lot of time, so I used my DVD-Key to download the Flight-Simulator via the Microsoft-Store. Since I did not have sufficient space on my internal HDD, I installed the game on my external SSD, which works out fine with the regular A320-model. 

Now I'm unsure about the correct installation-folder. Actually i pasted the "A32NX"-**Folder** (not only its content, but the folder as a whole) into **both** of the following directories at the same time;

1.: C:\Users\[USER]\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalCache\Packages\Community

and on my external SSD:

2. E:\Spiele\FS\Community

I've also activated the developer-mode. Nevertheless, the Mod is not working. As a test I tried to open the APU-Info-page. Does not work. 

I've heard that several people had problems using custom directories. Thus, there might be a solution by now. 

Kind regards
Max

**Comment by akcug** [comment]
This same problem. Game installed by game pass on internal second ssd disc E:\ Tried installer (installed files in community on disc E:\) but game did not load custom aircraft, still visible a320neo, after that tried copy A32NX folder to c:\...appdata...community and stil the same. Game must be installed in typical folder to run this mod? Thanks for any help.

**Comment by ghost** [comment]
I highly reccomend downloading and using our installer. It will install the mod to correct community folder. You can download it here: https://api.flybywiresim.com/installer

**Reply by akcug** [reply]
Ok, after third reinstall it works. First, it looks like problem witch custom install path, so after delete all content nad second reinstall in default patch C;/...appdata...pakages... the MSFS2020 crashed in start with installed A32nx (worked fine without mod). So again deleted all content, cleared registry, installed in default C;/...appdata...pakages... - cleared config file like: **https://flightsimulator.zendesk.com/hc/en-us/articles/360016763120--Only-2-aircraft-in-the-hangar-How-to-install-additional-content-First-installation-or-reinstallation-**
and installed a32nx with flybywire installer. Then everything looks work fine.
